tests/installation/ipxe_install.pm

```diff
@@ -104,21 +104,23 @@ sub set_bootscript {
     }
     $cmdline_extra .= " plymouth.enable=0 ";
 
-    # Extra options for virtualization tests with ipmi backend
-    if (get_var('VIRT_AUTOTEST') || get_var('HANA_PERF')) {
-        $cmdline_extra .= " video=1024x768 vt.color=0x07 " if check_var('VIDEOMODE', 'text');
-        # Support either IPXE_CONSOLE=ttyS1,115200 or SERIALDEV=ttyS1
-        my $serial_dev;
-        if (get_var('IPXE_CONSOLE')) {
-            get_var('IPXE_CONSOLE') =~ /^(\w+)/;
-            $serial_dev = $1;
-        }
-        else {
-            $serial_dev = get_var('SERIALDEV', 'ttyS1');
-            $cmdline_extra .= " console=$serial_dev,115200 ";
-        }
-        $cmdline_extra .= " Y2DEBUG=1 linuxrc.log=/dev/$serial_dev linuxrc.core=/dev/$serial_dev linuxrc.debug=4,trace reboot_timeout=0";
+    $cmdline_extra .= " video=1024x768 vt.color=0x07 " if check_var('VIDEOMODE', 'text');
+    # Support either IPXE_CONSOLE=ttyS1,115200 or SERIALDEV=ttyS1
+    my $serial_dev;
+    if (get_var('IPXE_CONSOLE')) {
```

Comment:
this if state need change. when it was for VIRT_AUTOTEST, it is ok. but now it is duplicate with line #93.

Reply:
I could change the condition to `if ($console)` but the code below just sets `$serial_dev` which is used differently from `$console`.

Reply:
`SERIALDEV` is used for VIRT_AUTOTEST only, we always set SERIALDEV=ttyS1 or SERIALDEV=ttyS2 in workers.ini. So you can keep the if statement for VIRT_AUTOTEST only. 

if `IPXE_CONSOLE` is set, `SERIALDEV` is useless.  They are similar. `if ($console)` is ok.

Reply:
The verification run has `SERIALDEV` but no `IPXE_CONSOLE`.

Reply:
Here is the case from VIRT_AUTOTEST tests: https://openqa.suse.de/tests/12135526. You can verify.

Reply:
Verify what? We have YaST tests which now use IPXE and define only `SERIALDEV`. Then we have kernel tests which define `IPXE_CONSOLE`. Both branches of the condition are needed globally.

Reply:
oow, I just got your meaning. Yes, your change is correct. I overlooked. Since `SERIALDEV` is added to worker settings, it has not been owned by VIRT_AUTOTEST only.